templates/login.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    {% block content %}
+    <h1>Login</h1>
+    <form method="POST" action="{{ url_for('login') }}">
+    {{ form.csrf_token }}
+    <p>
+    {{ form.first_name.label }}<br>
+    {{ form.first_name(size=32) }}
+    </p>
+        <p>
+    {{ form.second_name.label }}<br>
+    {{ form.second_name(size=32) }}
+    </p>
+        <p>
+    {{ form.email.label }}<br>
+    {{ form.email(size=32) }}
+    </p>
+    <p>
+    {{ form.password.label }}<br>
+    {{ form.password(size=32) }}
+    </p>
+    <p>
+    <input type="submit" value="Login">
+    </p>
```

Comment:
А тут у вас много повторений кода? Можно сделать так
```suggestion
        {% for field in form if field.name != 'csrf_token' %}
        <p><strong>field.label</strong> {{ field(size=20) }}</p>
        <ul class="alert alert-danger">
	    {% for error in field.errors %}
	    <li>{{ error }}</li>
	    {% endfor %}
	</ul>
        {% endfor %}
        <p>{{ form.submit() }}</p>
```
Причем строки 
```html
        <ul class="alert alert-danger">
	    {% for error in field.errors %}
	    <li>{{ error }}</li>
	    {% endfor %}
	</ul>
```
Повторяют сообщения flash об ошибках. Можно оставить что-то одно. Решать вам, что и где использовать
```